packages/plantuml-lsp/package.yaml

```
@@ -0,0 +1,18 @@
+---
+name: plantuml-lsp
+description: Language server for PlantUML.
+homepage: https://github.com/ptdewey/plantuml-lsp
+licenses:
+  - MIT
+languages:
+  - PlantUML
+categories:
+  - LSP
+
+source:
+  id: pkg:github/ptdewey/plantuml-lsp@v0.4.0
+  build:
+    run: go build
```

```suggestion
  id: pkg:golang/github.com/ptdewey/plantuml-lsp@v0.4.0

schemas:
  lsp: vscode:https://raw.githubusercontent.com/ptdewey/planuml-lsp/{{version}}/editors/code/package.json
```

We should use the `golang` installer rather than manually calling `go build`

This also recommends adding in the VS Code extension schema. This won't work until it is merged in (I see you have an open PR) but is for the future.

---

Oh that does seem nice, I might have to check over my package definition to ensure everything is well documented there (the VSCode extension was kind of an afterthought that I haven't put a lot of time into thus far).

I didn't see anything about the `schemas` option in the documentation, maybe it would be worth adding there somewhere?